packages/components/src/components/popup/popup.stories.ts

```diff
@@ -1,18 +1,37 @@
 import '../../solid-components';
-import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 import { withActions } from '@storybook/addon-actions/decorator';
 
-const { argTypes, args, parameters } = storybookDefaults('sd-popup');
+const { argTypes, parameters } = storybookDefaults('sd-popup');
 const { generateTemplate } = storybookTemplate('sd-popup');
+const { overrideArgs } = storybookHelpers('sd-popup');
 
 /**
  * Popup is a utility that lets you declaratively anchor "popup" containers to another element.
+ *
+ * This component’s name is inspired by <popup>. It uses Floating UI under the hood to provide a well-tested, lightweight, and fully declarative positioning utility for tooltips, dropdowns, and more.
+ *
+ * Popup doesn’t provide any styles — just positioning! The popup’s preferred placement, distance, and skidding (offset) can be configured using attributes.
+ *
+ * > Popup is a low-level utility built specifically for positioning elements. Do not mistake it for a tooltip or similar because it does not facilitate an accessible experience! Almost every correct usage of <sl-popup> will involve building other components. It should rarely, if ever, occur directly in your HTML.
  */
 
 export default {
-  title: 'Components/sd-popup',
+  title: 'Utilities/sd-popup',
```

Comment:
is this supposed to look like this?
![image](https://github.com/solid-design-system/solid/assets/48211526/8b0c54d1-6d2c-45a3-b1d3-78025e118c01)

Reply:
Yes. This is unstyled on purpose as this is just an unstyled utility component.

Reply:
I understand but can we move it at least to be completely visible in this story or is it too complicated? looks broken when opening it.

Reply:
Ah, got it. Yeah, sure!